utils/rsvp.js

```diff
@@ -0,0 +1,27 @@
+import {
+  Player,
+  Game,
+  Location,
+  GamePlayer,
+  Rsvp,
+  SkillLevel,
+} from "../src/models";
+
+async function rsvp(gameId, playerId, newRsvp) {
+  const original = await DataStore.query(GamePlayer, (c) => c.and(c => [
+    c.game_id.eq(gameId),
+    c.player_id.eq(playerId)
+  ]));
+
+  if (original && original.rsvp != newRsvp) {
```

Comment:
Would this only update an RSVP if one was made already? What about when someone RSVP's for the first time?

Reply:
It would update if the user selects an RSVP option that is different than what is currently in the database, so if the user accpeted, and they try to accept again, no point in making another write to the db

Reply:
Yes, but what if someone RSVPs for the first time? If that is the case, the query would return nothing (?) and the if statement would never execute, meaning someone RSVPing for the first time would not have their status updated. RSVPing can be in the next sprint if we want - if this is the case, comment this and replace with a console log or something.